test/export/fake-subscriber/Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM --platform=$BUILDPLATFORM golang:1.24-bookworm@sha256:fa1a01d362a7b9df68b021d59a124d28cae6d99ebd1a876e3557c4dd092f1b1d AS builder
```

Comment:
this will prob get bumped by dependabot to use the latest and to avoid CVEs. hence my question that maybe we would need to publish these in the future like all other GK images.

Reply:
+1, I agree on publishing images as well.